app.js

```diff
@@ -237,24 +326,22 @@ async function watchRandomStreamers(browser, page) {
     }
 
     // Is this streamer still streaming?
-    const channelStatus = await page
-      .$eval(CHANNEL_STATUS_QUERY, (el) => el.textContent)
-      .catch(() => "");
+    const channelStatus = (await query(page, CHANNEL_STATUS_QUERY)).text().trim().toUpperCase();
```

Comment:
Does this line work? I removed `query()`

Reply:
yes it works, don't remove anything

Reply:
I don't mean that I removed it in this line, I mean that I removed the `query()` function completely a few commits ago